static/src/javascripts/projects/common/views/svg.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+export const addClassesAndTitle = (
```

Comment:
Isn't that more what https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/views/svgs.js mimics, which then delegates to this?